editor/components/inserter/menu.js

```diff
@@ -16,7 +16,7 @@ function InserterMenu() {
 	const categories = wp.blocks.getCategories();
 
 	return (
-		<div className="editor-inserter__menu">
+		<div className={ `editor-inserter__menu is-${ position }` }>
```

Comment:
Should we have used `classnames` here? I guess it's a simple enough concatenation.